optin/reporting/utils/ServerUtils.js

```diff
@@ -0,0 +1,85 @@
+const moment = require('moment-timezone');
+
+const ServerUtils = {
+	/**
+	 * Convert amount to relative amount (6 divisibility).
+	 * @param {string|number} amount amount.
+	 * @returns {number|undefined} relative amount.
+	 */
+	toRelativeAmount: amount => {
+		const number = parseInt(amount, 10);
+
+		if (Number.isNaN(number))
+			return undefined;
+
+		return amount / (10 ** 6);
+	},
+	/**
+	 * Convert byte to hex string.
+	 * @param {string|Uint8Array} uint8arr Uint8Array or string.
+	 * @returns {string} hex string.
+	 */
+	byteToHexString: uint8arr => {
+		if (!uint8arr)
+			return '';
+
+		let hexStr = '';
+		for (let i = 0; i < uint8arr.length; i++) {
+			let hex = (uint8arr[i] & 0xff).toString(16);
+			hex = (1 === hex.length) ? `0${hex}` : hex;
+			hexStr += hex;
+		}
+
+		return hexStr.toUpperCase();
+	},
+	/**
+	 * Convert hex string to byte.
+	 * @param {string} str hex string.
+	 * @returns {Uint8Array} Uint8Array.
+	 */
+	hexStringToByte: str => {
+		if (!str)
+			return new Uint8Array();
+
+		const a = [];
+		for (let i = 0, len = str.length; i < len; i += 2)
+			a.push(parseInt(str.substr(i, 2), 16));
+
+		return new Uint8Array(a);
+	},
```

Comment:
we have similar functions in sdk-javascript, but we don't currently export them. maybe we should?

Reply:
yes, we should do it, I believe it will benefit other projects as well.

Reply:
please open issue against sdk to expose 👍

Reply:
https://github.com/symbol/symbol/issues/215